.pre-commit-config.yaml

```diff
@@ -166,6 +166,7 @@
       (?x)^(
         aiida/cmdline/commands/.*|
         aiida/cmdline/params/.*|
+        aiida/cmdline/params/types/.*|
```

Comment:
what pre-commit errors happened here? Can we get rid of this?

Reply:
this is the configuration of pre-commit, and this particular section defines which files to watch for updating the automatic documentation of the verdi cli.

Since the "metavar" parameter depends on the param types, we need to watch these files as well.